operators/src/pro/ml/xgboost.rs

```diff
@@ -0,0 +1,916 @@
+            byte_size_ax_0,
+            byte_size_ax_1,
+            25,
+            3,
+            -1,
+            f32::NAN,
+        )
+        .unwrap();
+
+        xg_matrix
+    }
+
+    #[tokio::test]
+    async fn multi_tile_test() {
+        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
+        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
+        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
+        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
+        // 255, 255,   0,   0,   0  ||  0,   0,   0, 255, 255
+        let green = make_double_raster(
+            vec![
+                255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255,
+                255, 0, 0, 0,
+            ],
+            vec![
+                0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0,
+                0, 255, 255,
+            ],
+        );
+
+        //   0,   0,   0, 255, 255  ||  255, 255,   0,   0,   0
+        //   0,   0,   0, 255, 255  ||  255, 255,   0,   0,   0
+        //   0,   0,   0,   0,   0  ||    0,   0,   0,   0,   0
+        //   0,   0,   0, 255, 255  ||  255, 255,   0,   0,   0
+        //   0,   0,   0, 255, 255  ||  255, 255,   0,   0,   0
+        let blue = make_double_raster(
+            vec![
+                0, 0, 0, 255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 0, 0, 0, 0, 0, 255, 255, 0, 0, 0,
+                255, 255,
+            ],
+            vec![
+                255, 255, 0, 0, 0, 255, 255, 0, 0, 0, 0, 0, 0, 0, 0, 255, 255, 0, 0, 0, 255, 255,
+                0, 0, 0,
+            ],
+        );
+
+        //   0,   0, 255,   0,   0  ||     0,   0, 255,   0,   0
+        //   0,   0, 255,   0,   0  ||     0,   0, 255,   0,   0
+        //   0,   0, 255, 255, 255  ||   255, 255,   0,   0,   0
+        //   0,   0, 255,   0,   0  ||     0,   0, 255,   0,   0
+        //   0,   0, 255,   0,   0  ||     0,   0, 255,   0,   0
+        let temp = make_double_raster(
+            vec![
+                0, 0, 255, 0, 0, 0, 0, 255, 0, 0, 0, 0, 255, 255, 255, 0, 0, 255, 0, 0, 0, 0, 255,
+                0, 0,
+            ],
+            vec![
+                0, 0, 255, 0, 0, 0, 0, 255, 0, 0, 255, 255, 255, 0, 0, 0, 0, 255, 0, 0, 0, 0, 255,
+                0, 0,
+            ],
+        );
+
+        let srcs = vec![green, blue, temp];
+
+        let model_path = PathBuf::from("pro/ml/xgboost/s2_10m_de_marburg/model.json");
+
+        let xg = XgboostOperator {
+            params: XgboostParams {
+                model_sub_path: model_path.clone(),
+                no_data_value: f32::NAN,
+            },
+            sources: MultipleRasterSources { rasters: srcs },
+        };
+
+        let mut exe_ctx = MockExecutionContext::new_with_tiling_spec(TilingSpecification::new(
+            (0., 0.).into(),
+            [5, 5].into(),
+        ));
+
+        exe_ctx.initialize_ml_model(model_path);
```

Comment:
please unwrap the result so we now where it failed

Reply:
https://github.com/geo-engine/geoengine/commit/fd940cc820ae7165086bbe9e41f6af5fedf83830